src/Conventional_Monitor/image_gen/ush/plot_horz.sh

```diff
@@ -35,23 +31,12 @@
 
```

Comment:
Line 26-30, the location of the anl and ges directories are in `${hh_tankdir}/horz_hist`.

Reply:
`hh_tankdir` was being set to `${C_TANKDIR}/${RUN}.${PDY}/${CYC}/conmon/horz_hist` at line 10.  However that should be replaced by using `get_stats_path.sh` in order to support alternate data storage schemes.  I've made that change as well as added a check on compressed analysis and guess files prior to trying to uncompress them.